data_steward/resource_files/schemas/wearables/fitbit/activity_summary.json

```diff
@@ -76,5 +76,11 @@
     "name": "person_id",
     "mode": "nullable",
     "description": ""
+  },
+  {
+    "type": "string",
+    "name": "src_id",
+    "mode": "nullable",
+    "description": "This id functions as a denotation to signify the provenance of this entry"
```

Comment:
Nit: Re-word the description to align with the program language.

`Flag indicating the participant portal that this record originated from. `